mtpc/model_utils.py

```diff
@@ -181,26 +183,37 @@ def sample_standard_gaussian(mu, sigma):
 
 
 def predict(encoder, ode_func, classifier, tol, latent_dim, ptnm, times, features, cmax_time, device="cpu"):
+    # generate dosing information to integrate into the latent features that go into the ODE solver
     dosing = torch.zeros([features.size(0), features.size(1), latent_dim])
     dosing[:, :, 0] = features[:, :, -2]
     dosing = dosing.permute(1, 0, 2).to(device)
 
+    # get encoder output and sample latent features from a gaussian latent
```

Comment:
We may want to cover why there is a latent distribution and what role it serves.